native/src/modules/app/components/AppStateListener.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import { useEffect } from 'react'
+import { AppState } from 'react-native'
+import sendTrackingSignal from '../../endpoint/sendTrackingSignal'
+import { RESUME_SIGNAL_NAME, SUSPEND_SIGNAL_NAME } from 'api-client'
+
+const AppStateListener = () => {
+  useEffect(() => {
+    AppState.addEventListener('change', handleAppStateChange)
+
+    return () => {
+      AppState.removeEventListener('change', handleAppStateChange)
+    }
+  }, [])
+
+  const handleAppStateChange = (nextAppState: string) => {
+    if (nextAppState === 'active') {
```

Comment:
there is also inactive on iOS: https://reactnative.dev/docs/appstate#app-states

I think this should also be tested on iOS and Android.